cjapy/cjapy.py

```diff
@@ -2560,3 +2560,256 @@ def clean_column_name(col):
         
         df.columns = [clean_column_name(col) for col in df.columns]
         return df
+
+    def getPersonProfiles(
+        self,
+        dataviewId: str = None,
+        personId: str = "variables/adobe_identitynamespace_personid",
+        featureMetrics: List[str] = None,
```

Comment:
@admoseremic 
FYI : The typing such as `List[str]` is only supported above a certain version of python, so I usually remove them before going live so the module can be used with python 3.6 and above (when it was not available). 
So you know why you may not see them.